### DISCUSSION TITLE
Can Bark or any other approach work real time?

### DISCUSSION THREAD

**Discussion by grexzen** [serp-ai/bark-with-voice-clone#7] [Q&A]
Looking to replace customer service calls by using this tech to answer questions from users.

Whisper, VallE, Eleven Labs are awesome, but have not seen any real time conversations. 

Any help? Tx!

**Comment by KhiryMcCurn** [comment]
Bark would be the output; however, you would need to use multiple tools to understand the customer's audio and then build the prompt to feed Bark.

**Reply by devinschumacher** [reply]
We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉Check it out: https://www.youtube.com/watch?v=84JVGtkapik
